app/store/slices/vaultUserSlice.ts

```diff
@@ -30,7 +30,8 @@ export const fetchVaultUserData = createAsyncThunk(
   },
 )
 
-interface VaultUserSlice {
+// TODO: add lint rule to enforce export all interfaces
```

Comment:
rip tsserver lsp

Reply:
I think it only will improve it. We have bunch of duplicated types that have same stucture